app/scripts/views/sign_in.js

```diff
@@ -85,7 +85,9 @@ define(function (require, exports, module) {
 
     afterVisible: function () {
       FormView.prototype.afterVisible.call(this);
-      return this.displayAccountProfileImage(this.getAccount());
+      return this.displayAccountProfileImage(this.getAccount(),
+                                             null /* wrapper */,
```

Comment:
Ah, I see, `wrapperClass` is optional. Since that's the case, perhaps we should remove the param and add a new item to the `options` block, `wrapperClass`, to simplify the function signature slightly.